style.css

```diff
@@ -395,3 +429,407 @@ textarea:focus {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    margin-bottom: 20px;
+  }
+
+  .text2 {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .contacts {
+    display: flex;
+    flex-direction: column;
+    margin-top: 351px;
+  }
+}
+
+@media (min-width: 990px) {
+  .work-section {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    padding: 100px 100px;
+    background-image: none;
+  }
+
+  .MY-works {
+    display: flex;
+    gap: 52px;
+    margin-bottom: 30px;
+  }
+
+  .part1 {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+  }
+
+  h2 {
+    height: 55px;
+  }
+
+  .recent-work {
+    font-family: "Crete Round", sans-serif;
+    font-size: 40px;
+    text-align: left;
+    color: #091e42;
+  }
+
+  .line11 {
+    border: 1px solid rgba(33, 33, 33, 1);
+    height: 0;
+    flex: 1;
+    color: #212121;
+  }
+
+  .placeholder {
+    height: 448px;
+    width: 684px;
+  }
+
+  .mini-title {
+    font-family: "Crete Round", sans-serif;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .dummy-text {
+    color: rgba(9, 30, 66, 1);
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .css-container {
+    display: flex;
+    gap: 12px;
+    justify-content: flex-start;
+  }
+
+  .css {
+    color: rgba(9, 30, 66, 1);
+    padding: 12px 10px;
+    font-family: "Inter", sans-serif;
+    font-size: 15px;
+    border: 1px solid rgba(137, 147, 164, 1);
+    width: auto;
+  }
+
+  .invite-one {
+    font-family: "Inter", sans-serif;
+    font-size: 17px;
+    text-align: center;
+    color: rgba(255, 255, 255, 1);
+    padding: 8px;
+  }
+}
+
+@media (min-width: 990px) {
+  .professional-art {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-gap: 10px;
+  }
+
+  .art-container.one,
+  .art-container.four {
+    background-image: url(icons/ImgPlaceholder1.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .art-container.two,
+  .art-container.five {
+    background-image: url(icons/imgplace202.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .art-container.three,
+  .art-container.six {
+    background-image: url(icons/imageplaceholder303.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .art-container.one {
+    position: relative;
+  }
+
+  .art-container > .invite {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    display: none;
+  }
+
+  .art-item {
+    background-image: none;
+    padding-top: 150px;
+  }
+
+  .data {
+    font-family: "Create Round", sans-serif;
+    font-size: 32px;
+    line-height: 44px;
+    font-weight: 400;
+    color: rgba(255, 255, 255, 1);
+  }
```

Comment:
- [ ] Kindly follow the color of this heading `<h2>` in the mobile version. check the difference here 👇  

Your design | The template 
--| --
![image](https://user-images.githubusercontent.com/65068771/149509824-a374654e-86c0-4d9c-a410-05388352e4cf.png) | ![image](https://user-images.githubusercontent.com/65068771/149509900-e00eee96-001c-4bc9-9da8-8953dd58fca3.png)